pkg/reconciler/pipelinerun/controller.go

```diff
@@ -50,6 +51,7 @@ func NewController(ctx context.Context, cmw configmap.Watcher) *controller.Impl
 		PipelineRunSigner: psSigner,
```

Comment:
Try to put the metrics recorder inside this signer struct.

Reply:
Done